src/plugins/intel_cpu/tests/functional/shared_tests_instances/behavior/ov_plugin/core_integration.cpp

```diff
@@ -87,7 +87,19 @@ INSTANTIATE_TEST_SUITE_P(
 const std::vector<ov::AnyMap> multiConfigs = {
     {ov::device::priorities(CommonTestUtils::DEVICE_CPU)}
 };
-const std::vector<ov::AnyMap> configsDeviceProperties = {{ov::device::properties("CPU", ov::num_streams(3))}};
+const std::vector<ov::AnyMap> configsDeviceProperties = {{ov::device::properties("CPU", ov::num_streams(3))},
+                                                         {ov::device::properties(ov::AnyMap{{"CPU", ov::AnyMap{ov::num_streams(3)}}})}};
+const std::vector<ov::AnyMap> configsDevicePropertiesDouble = {{ov::device::properties("CPU", ov::num_streams(5)),
+                                                                ov::num_streams(3)},
+                                                               {ov::device::properties("CPU", ov::num_streams(5)),
+                                                                ov::device::properties(ov::AnyMap{{"CPU", ov::AnyMap{ov::num_streams(7)}}}),
+                                                                ov::num_streams(3)},
```

Comment:
Do we support multiple ov::device::properties items into one compile_model() calling? @nkogteva

Reply:
> Do we support multiple ov::device::properties items into one compile_model() calling? @nkogteva

On current master user can specified `ov::device::properties("CPU", ov::num_streams(3)), ov::device::properties("CPU", ov::num_streams(5)` and it will be resolved on API level. Only one propery will be kept.

But on the current branch user can specified property in different ways. And it will be not resolved on API level, because keys are different. So we decided to keep first priority for form `ov::device::properties("DEVICE", ...)`. For example, user can use some map for different devices and override some specific property for some specific device in compile model instead of updating map.